src/ittnotify/jitprofiling.c

```diff
@@ -112,6 +114,28 @@ ITT_EXTERN_C iJIT_IsProfilingActiveFlags JITAPI iJIT_IsProfilingActive()
     return executionMode;
 }
 
+#if ITT_PLATFORM==ITT_PLATFORM_WIN
+int isPathRelative(char *path)
+{
+    if(path==NULL)
+    {
+        return 1;
+    }
+    else if(strlen(path)>1) 
+    {
+        if(isalpha(path[0]) && path[1]==':')
```

Comment:
Lets add check `path[2] == '\\'` to confirm that this path is an absolute and valid

Reply:
If I am understanding this correctly. We should add one more check for `path[2] == '\\'` over `isalpha(path[0]) && path[1]==':'`  then return it's an absolute path. In that case, I feel we might might misclassify paths such as `D:` or `C:` which are absolute paths.
Please let me know if I am misinterpreting the comment or missing something.

Reply:
If the path satisfies all these 3 conditions, then we decide that it is an absolute path

Reply:
Have added the condition in latest pushed changes.